src/BottomNavigation/BottomNavigationItem.js

```diff
@@ -0,0 +1,89 @@
+// @flow weak
+
+import React, { PropTypes, cloneElement } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+import ButtonBase from '../internal/ButtonBase';
+
+export const styleSheet = createStyleSheet('BottomNavigationItem', (theme) => {
+  const { palette, typography } = theme;
+
+  return {
+    root: {
+      transition: 'color 0.2s, padding-top 0.2s',
```

Comment:
couple of things:

1. We should be using the transition function on theme as it uses the material design specified easing function
2. Why not use flexbox to align the content?